cgra/test/CgraWithContextSwitchRTL_test.py

```diff
@@ -874,9 +874,21 @@ def sim_fir_return(cmdline_opts, mem_access_is_combinational):
         IntraCgraPktType(0, 8, payload = CgraPayloadType(CMD_TERMINATE)),
         IntraCgraPktType(0, 9, payload = CgraPayloadType(CMD_TERMINATE))
       ],
+      
+      # Terminate double time to make sure all registers are cleared
+      [
+        IntraCgraPktType(0, 0, payload = CgraPayloadType(CMD_TERMINATE)),
+        IntraCgraPktType(0, 1, payload = CgraPayloadType(CMD_TERMINATE)),
+        IntraCgraPktType(0, 4, payload = CgraPayloadType(CMD_TERMINATE)),
+        IntraCgraPktType(0, 5, payload = CgraPayloadType(CMD_TERMINATE)),
+        IntraCgraPktType(0, 8, payload = CgraPayloadType(CMD_TERMINATE)),
+        IntraCgraPktType(0, 9, payload = CgraPayloadType(CMD_TERMINATE))
+      ],
```

Comment:
You just want to make the clock tick a few cycles, right? Why immediate resume doesn't work?